MANIFEST.in

```diff
@@ -1,2 +1,5 @@
 recursive-include pulse/templates *
 recursive-include pulse/static *
+recursive-include data/ineligible *
```

Comment:
This can likely be 🔥 at some later point, am just including it as the current system breaks without it.

Reply:
yeah, agreed with the 🔥 since it's specific to 18Fs pulse, but in another PR